gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineErrorProneIntegrationTest.groovy

```diff
@@ -100,4 +101,28 @@ class BaselineErrorProneIntegrationTest extends AbstractPluginTest {
         BuildResult result = with('compileJava').build()
         result.task(":compileJava").outcome == TaskOutcome.SUCCESS
     }
+
+    def 'compileJava applies patches when error-prone finds errors'() {
+        when:
+        buildFile << standardBuildFile
+        file('src/main/java/test/Test.java') << inValidJavaFile
+
+        then:
+        BuildResult result = with('compileJava', "-P${BaselineErrorProne.PROP_ERROR_PRONE_APPLY}=true").build()
```

Comment:
instead of using the constant, could you hard code the value in the test? it helps prevent accidental changes to the flag